addon/tracked.ts

```diff
@@ -90,15 +90,13 @@ function installTrackedProperty(_target: object, key: string | symbol, descripto
   }
 
   return {
-    configurable: true,
-    // TODO: correcting a misspelling caused chrome to error
-    // writable: true,
```

Comment:
FWIW this is intentional. I fixed the same problem in this PR with a detailed analysis: https://github.com/salsify/ember-css-modules/pull/118